### DISCUSSION TITLE
Weight is not actually displaying the unit in product details --  Request for multiple units

### DISCUSSION THREAD

**Discussion by brbintenyears** [bagisto/bagisto#5166] [Ideas]
When you allow weight to be displayed on the product information page, it just displays the number without the actual unit. This can be very confusing for customers. It should be displayed. 

Also, I think there should be the ability to use multiple units. For example, there should be the option to specify the weight of one product in lbs but the weight of another product in oz. It would be very convenient.

Thanks.

**Comment by vaishaliwebkul** [comment]
@amrh910 
We can consider this as an enhancement and will be great if you could contribute for this feature.

**Comment by devansh-webkul** [comment]
- This can be achieved by creating attributes like `weight_in_lbs` => 'Weight (In LBS)', `weight_in_gram` => 'Weight (In Gram)'.
- Assigned those to different attribute family.
- Done!
